src/scrapers/hapoalim.js

```diff
@@ -24,6 +24,15 @@ function convertTransactions(txns) {
   });
 }
 
+function fetchSummary(balanceCreditResult) {
```

Comment:
@chilikla perhaps rename to `convertToSummary`? This function doesn't "fetch" anything.